### PR TITLE
Create pie charts for linkage errors using SVG

### PR DESCRIPTION
**Pull request by elharo**

### CODE REVIEW

File: dashboard/src/main/resources/templates/dashboard.ftl [hunk 1]
```diff
@@ -40,8 +41,26 @@
...
     </section>
     
+    <section class="piecharts">
+      <h3>Linkage Errors</h3>
+      
+      <#assign ratio = linkageErrorCount / totalArtifacts >
+      <#assign endPointX = pieChart.calculateEndPointX(100, 100, 100, ratio)>
+      <#assign endPointY = pieChart.calculateEndPointY(100, 100, 100, ratio)>
+      
+      <p>${linkageErrorCount} out of ${totalArtifacts} artifacts have linkage errors.</p>
```
Comment (suztomo) [inline dashboard/src/main/resources/templates/dashboard.ftl]: Subject-verb agreement when `linkageErrorCount` is 1.

"1 out of 187 artifacts have linkage errors." -> "1 out of 187 artifacts has linkage errors."

File: dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/PieChartTest.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    Assert.assertEquals(expected.getX(), actual.getX(), TOLERANCE);
+    Assert.assertEquals(expected.getY(), actual.getY(), TOLERANCE);
+  }
+ 
+  @Test
+  public void testFull() {
+    Point2D actual = PieChart.calculateEndPoint(100, 100, 100, 1.0);
+    Point2D expected = new Point2D.Double(100, 0);
+    Assert.assertEquals(expected.getX(), actual.getX(), TOLERANCE);
+    Assert.assertEquals(expected.getY(), actual.getY(), TOLERANCE);  }
```
Comment (suztomo) [inline dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/PieChartTest.java]: new line before `}`

Comment (elharo) [reply dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/PieChartTest.java]: done

File: dashboard/src/main/resources/templates/dashboard.ftl (hunk 1 above)
Comment (elharo) [reply dashboard/src/main/resources/templates/dashboard.ftl]: done

**Pull request merged**